src/runners/http_rfqm_service_runner.ts

```diff
@@ -96,6 +97,12 @@ export async function buildRfqmServiceAsync(connection: Connection, asWorker: bo
 
     const contractAddresses = await getContractAddressesForNetworkOrThrowAsync(provider, CHAIN_ID);
     const axiosInstance = Axios.create(getAxiosRequestConfig());
+    axiosInstance.defaults.raxConfig = {
+        retry: 3, // Retry on 429, 500, etc.
```

Comment:
Should we just retry on 5xx? a 4xx is considered to be a reasonable rejection to "last look" right?

Reply:
Yup, by default it retries all 5xx and 429 Too Many Requests

Reply:
oh I see, thnanks